validator/keymanager/remote-web3signer/metrics.go

```diff
@@ -50,6 +50,14 @@ var (
 		Name: "remote_web3signer_blinded_block_capella_sign_requests_total",
 		Help: "Total number of block capella sign requests",
 	})
+	blockDenebSignRequestsTotal = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "remote_web3signer_block_deneb_sign_requests_total",
+		Help: "Total number of block deneb sign requests",
+	})
+	blindedBlockDenebSignRequestsTotal = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "remote_web3signer_blinded_block_deneb_sign_requests_total",
+		Help: "Total number of block deneb sign requests",
```

Comment:
```suggestion
		Help: "Total number of blinded block deneb sign requests",
```

Same comment for `blindedBlockCapellaSignRequestsTotal` above